add Rocky to list of Redhat-dists

sircubbi opened the pull request:
Add Rocky to the list of Redhat-distributions to allow usage on a freshly installed Rocky-system.

nward commented:
Thanks @sircubbi - certainly need to get this added.

Can you also add this to manifest.json as part of your PR, and check if there are any tests which should be updated?

sircubbi commented:
There are no checks to update, but you are right, the manifest was quite outdated. Only Redhat 7 was listed there, so I added CentOS and Rocky as well, and also added version 8 to the list (which works fine as there is no difference between EL7 and EL8 in that regard).

Also I made sure too, that params.pp uses version 3 of freeradius on EL8.

nward commented:
I guess we'd better add some checks where the OS is relevant then!

Thanks - merging :-)
